internal/cluster/distribution/eks/eksworkflow/workflow_update_node_pool.go

```diff
@@ -160,6 +165,13 @@ func (w UpdateNodePoolWorkflow) Execute(ctx workflow.Context, input UpdateNodePo
 		if effectiveVolumeSize == 0 {
 			effectiveVolumeSize = parameters.NodeVolumeSize
 		}
+
+		securityGroups := parameters.CustomNodeSecurityGroups
```

Comment:
**Note**: I think this var could be avoided now that a strongly typed field is available.

Reply:
fixed